curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -17,439 +17,478 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
 
-Placeholder question
+Which attribute specifies that the element must be filled out before submitting the form?
 
 #### --distractors--
 
-Placeholder distractor 1
+`necessary`
 
 ---
 
-Placeholder distractor 2
+`imp`
 
 ---
 
-Placeholder distractor 3
+Not possible via HTML only
 
 #### --answer--
 
-Placeholder answer
+`required`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to hide an input from the user?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<hidden></hidden>`
 
 ---
 
-Placeholder distractor 2
+`<input type="text" hidden>`
 
 ---
 
-Placeholder distractor 3
+Not possible via HTML
 
 #### --answer--
 
-Placeholder answer
+`<input type="hidden" name="inputName" value="inputValue">`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `_target` attribute do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It specifies the variable we intend to change.
 
 ---
 
-Placeholder distractor 2
+It specifies different ways to store the response of the form.
 
 ---
 
-Placeholder distractor 3
+It specifies different keyword searches.
 
 #### --answer--
 
-Placeholder answer
+It specifies a name or keyword that indicates where to display the response after submitting the form.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following input type is correct?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<input type="search">`
 
 ---
 
-Placeholder distractor 2
+`<input type="range" min="" max="">`
 
 ---
 
-Placeholder distractor 3
+`<input type="email">`
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+`<form action="action.php" method="post">` When is post method used?
 
 #### --distractors--
 
-Placeholder distractor 1
+When receiving data to a script.
 
 ---
 
-Placeholder distractor 2
+When a user is receiving data.
 
 ---
 
-Placeholder distractor 3
+To post a the forms response.
 
 #### --answer--
 
-Placeholder answer
+When submitting data to a script.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to define the length of the input?
 
 #### --distractors--
 
-Placeholder distractor 1
+It is not possible.
 
 ---
 
-Placeholder distractor 2
+`<input length="15">`
 
 ---
 
-Placeholder distractor 3
+`<input len="15">`
 
 #### --answer--
 
-Placeholder answer
+`<input minlength="3" maxlength="15">`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to properly apply form validation to a form?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<input type="button" formnovalidate>`
 
 ---
 
-Placeholder distractor 2
+`<button type="button" formnovalidate></button>`
 
 ---
 
-Placeholder distractor 3
+`<input type="text" formnovalidate>`
 
 #### --answer--
 
-Placeholder answer
+`<button type="submit" formnovalidate></button>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following `<input>` type attribute is incorrect?
```

Comment:
this is a repeat of a previous question

Reply:
It's similar but the not the exact same question.